homeassistant/components/point/config_flow.py

```diff
@@ -0,0 +1,197 @@
+"""Config flow for Minut Point."""
+import asyncio
+from collections import OrderedDict
+import logging
+
+import async_timeout
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components.http import HomeAssistantView
+from homeassistant.core import callback
+
+DOMAIN = 'point'
+CLIENT_ID = 'client_id'
+CLIENT_SECRET = 'client_secret'
+KEY_IMPORT_GROUPS = 'import_groups'
+
+AUTH_CALLBACK_PATH = '/api/minut'
+AUTH_CALLBACK_NAME = 'api:minut'
+
+DATA_FLOW_IMPL = 'point_flow_implementation'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@callback
+def register_flow_implementation(hass, domain, name, client_id, client_secret):
+    """Register a flow implementation.
+
+    domain: Domain of the component responsible for the implementation.
+    name: Name of the component.
+    client_id: Client id.
+    client_secret: Client secret.
+    """
+    if DATA_FLOW_IMPL not in hass.data:
+        hass.data[DATA_FLOW_IMPL] = OrderedDict()
+
+    hass.data[DATA_FLOW_IMPL][domain] = {
```

Comment:
Why is the domain a variable and not a constant in this function? Only this component will call this function.

Reply:
Because that is how @balloob [like to have it](https://github.com/home-assistant/home-assistant/pull/17466#discussion_r230202913) for future use when the keys are stored in the home assistant cloud.